src/Microsoft.Management.Deployment/PackageManager.idl

```diff
@@ -885,7 +885,65 @@ namespace Microsoft.Management.Deployment
         String DocumentUrl { get; };
     }
 
-    /// IMPLEMENTATION NOTE: SourceAgreement from AppInstaller::Repository::SourceAgreement
+    /// Icon resolution
+    [contract(Microsoft.Management.Deployment.WindowsPackageManagerContract, 6)]
+    enum IconResolution
+    {
+        Unknown,
+        Custom,

```

Comment:
Isn't `Custom` just an official way of saying `Unknown`?  Should we get rid of `Unknown` and just return `Custom` instead?

Reply:
Had offline discussion with John. From caller perspective, Custom or Unknown both indicates "caller, figure it out by yourself".

"But still, "known to to you but not to me" is no different than "known to no one" if my code needs to handle strange sizes"